src/index.html

```diff
@@ -8,6 +8,54 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+
+    <h1 class="stars stars--0">
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+    </h1>
+
+    <h1 class="stars stars--1">
+      <div class="stars__star active"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+    </h1>
+
+    <h1 class="stars stars--2">
+      <div class="stars__star active"></div>
+      <div class="stars__star active"></div>
```

Comment:
remove class active, and use [nth-child](https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-child)